src/app/app-interactive-map/store/interactive-map.effects.ts

```diff
@@ -36,8 +37,29 @@ const preferredMaps = [
 const preferredMap = (mapItems: MapItem[]): MapItem =>
   mapItems.find((mapItem) => preferredMaps.includes(mapItem.name)) || mapItems[0];
 
+const preferredSpecies = [
+  'Escherichia coli',
+];
+
+const preferredSpecie = (species: Specie[]): Specie =>
+  species.find((speciesItem) => preferredSpecies.includes(speciesItem.name)) || species[0];
+
 @Injectable()
 export class InteractiveMapEffects {
+  @Effect()
+  fetchSpecies: Observable<Action> = this.actions$.pipe(
+    ofType(fromActions.FETCH_SPECIES),
+    switchMap((action: fromActions.FetchSpecies) => {
+      return this.http.get(`${environment.apis.warehouse}/organisms`);
+    }),
+    map((payload: Specie[]) => new fromActions.SetSpecies(payload)),
+  );
+
+  @Effect()
+  setSpecies: Observable<Action> = this.actions$.pipe(
+    ofType(fromActions.SET_SPECIES),
+    map((action: fromActions.SetSpecies) => new SetSelectedSpecies(preferredSpecie(action.payload).id)));
```

Comment:
Maybe you can make preferredItem more simple by working with list of strings instead list of objects.